2022-02-02 03:10:22,382 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-02-02 03:10:22,383 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /work/wandb/settings
2022-02-02 03:10:22,383 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-02 03:10:22,383 INFO    MainThread:71 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-02 03:10:22,383 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:10:22,383 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:10:22,383 INFO    MainThread:71 [wandb_init.py:_log_setup():386] Logging user logs to /work/wandb/run-20220202_031022-xgb_depth_test/logs/debug.log
2022-02-02 03:10:22,384 INFO    MainThread:71 [wandb_init.py:_log_setup():387] Logging internal logs to /work/wandb/run-20220202_031022-xgb_depth_test/logs/debug-internal.log
2022-02-02 03:10:22,384 INFO    MainThread:71 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f1ae84732d0>
2022-02-02 03:10:22,384 INFO    MainThread:71 [wandb_init.py:init():420] calling init triggers
2022-02-02 03:10:22,384 INFO    MainThread:71 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-02-02 03:10:22,385 INFO    MainThread:71 [wandb_init.py:init():442] re-initializing run, found existing run on stack: xgb_depth_test
2022-02-02 03:10:22,392 INFO    MainThread:71 [wandb_run.py:finish():1468] finishing run cz4041/test-project/xgb_depth_test
2022-02-02 03:10:22,392 INFO    MainThread:71 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-02 03:10:22,395 INFO    MainThread:71 [wandb_run.py:_restore():1769] restore
2022-02-02 03:10:22,811 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:10:22,984 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:10:23,323 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 28220
}

2022-02-02 03:10:23,438 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 28220
}

2022-02-02 03:10:23,543 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5123
  total_bytes: 28220
}

2022-02-02 03:10:23,646 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 28220
  total_bytes: 28220
}

2022-02-02 03:10:23,748 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 28220
  total_bytes: 28220
}

2022-02-02 03:10:23,850 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 28220
  total_bytes: 28220
}

2022-02-02 03:10:23,952 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 28220
  total_bytes: 28220
}

2022-02-02 03:10:24,054 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 28220
  total_bytes: 28220
}

2022-02-02 03:10:24,318 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 28220
  total_bytes: 28220
}
local_info {
}

2022-02-02 03:10:25,979 INFO    MainThread:71 [wandb_run.py:_append_files():2194] logging synced files
2022-02-02 03:10:25,997 INFO    MainThread:71 [wandb_init.py:init():471] starting backend
2022-02-02 03:10:25,997 INFO    MainThread:71 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-02 03:10:26,001 INFO    MainThread:71 [backend.py:ensure_launched():219] starting backend process...
2022-02-02 03:10:26,013 INFO    MainThread:71 [backend.py:ensure_launched():225] started backend process with pid: 1630
2022-02-02 03:10:26,018 INFO    MainThread:71 [wandb_init.py:init():480] backend started and connected
2022-02-02 03:10:26,031 INFO    MainThread:71 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-02-02 03:10:26,031 INFO    MainThread:71 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-02 03:10:26,031 INFO    MainThread:71 [wandb_init.py:init():550] updated telemetry
2022-02-02 03:10:26,044 INFO    MainThread:71 [wandb_init.py:init():581] communicating current version
2022-02-02 03:10:28,678 INFO    MainThread:71 [wandb_init.py:init():586] got version response 
2022-02-02 03:10:28,678 INFO    MainThread:71 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-02 03:10:28,805 INFO    MainThread:71 [wandb_init.py:init():624] starting run threads in backend
2022-02-02 03:10:33,828 INFO    MainThread:71 [wandb_run.py:_console_start():1827] atexit reg
2022-02-02 03:10:33,831 INFO    MainThread:71 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-02-02 03:10:33,831 INFO    MainThread:71 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-02-02 03:10:33,844 INFO    MainThread:71 [wandb_run.py:_redirect():1762] Redirects installed.
2022-02-02 03:10:33,845 INFO    MainThread:71 [wandb_init.py:init():651] run started, returning control to user process
2022-02-02 03:10:33,847 INFO    MainThread:71 [wandb_run.py:_config_callback():966] config_cb None None {'boosting_type': 'dart', 'objective': 'multi:softmax', 'num_class': 9, 'max_depth': 2, 'min_child_weight': 20, 'gamma': 1, 'subsample': 0.8, 'colsample_bytree': 0.7, 'tree_method': 'hist', 'eval_metric': 'mlogloss', 'eta': 0.04, 'alpha': 1, 'verbose': 2}
2022-02-02 03:10:34,156 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:10:34,156 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:10:34,208 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:10:34,383 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:10:34,383 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:11:09,058 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:11:09,061 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:11:09,063 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:11:09,083 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:11:09,247 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:11:09,247 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:11:12,329 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:11:12,331 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:11:12,332 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:11:12,347 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:11:12,475 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:11:12,475 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:11:15,073 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:11:15,202 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-02-02 03:11:15,202 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /work/wandb/settings
2022-02-02 03:11:15,202 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-02 03:11:15,203 INFO    MainThread:71 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-02 03:11:15,203 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:11:15,203 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:11:15,203 INFO    MainThread:71 [wandb_init.py:_log_setup():386] Logging user logs to /work/wandb/run-20220202_031115-xgb_depth_test/logs/debug.log
2022-02-02 03:11:15,203 INFO    MainThread:71 [wandb_init.py:_log_setup():387] Logging internal logs to /work/wandb/run-20220202_031115-xgb_depth_test/logs/debug-internal.log
2022-02-02 03:11:15,203 INFO    MainThread:71 [wandb_init.py:init():420] calling init triggers
2022-02-02 03:11:15,204 INFO    MainThread:71 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-02-02 03:11:15,204 INFO    MainThread:71 [wandb_init.py:init():442] re-initializing run, found existing run on stack: xgb_depth_test
2022-02-02 03:11:15,211 INFO    MainThread:71 [wandb_run.py:finish():1468] finishing run cz4041/test-project/xgb_depth_test
2022-02-02 03:11:15,246 INFO    MainThread:71 [jupyter.py:save_history():458] saving 18 cells to _session_history.ipynb
2022-02-02 03:11:15,285 INFO    MainThread:71 [wandb_run.py:_config_callback():966] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2022-02-02 03:11:15,356 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:11:15,356 INFO    MainThread:71 [wandb_init.py:_jupyter_teardown():318] cleaning up jupyter logic
2022-02-02 03:11:15,357 INFO    MainThread:71 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-02 03:11:15,359 INFO    MainThread:71 [wandb_run.py:_restore():1769] restore
2022-02-02 03:11:15,964 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:11:16,865 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:11:17,546 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 28268
}

2022-02-02 03:11:17,658 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 65076
}

2022-02-02 03:11:17,769 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 65076
}

2022-02-02 03:11:17,871 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 6052
  total_bytes: 65076
}

2022-02-02 03:11:17,974 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 6052
  total_bytes: 65076
}

2022-02-02 03:11:18,076 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 6052
  total_bytes: 65076
}

2022-02-02 03:11:18,195 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 6052
  total_bytes: 65076
}

2022-02-02 03:11:18,300 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 65076
  total_bytes: 65076
}

2022-02-02 03:11:18,402 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 65076
  total_bytes: 65076
}

2022-02-02 03:11:18,505 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 65076
  total_bytes: 65076
}

2022-02-02 03:11:18,607 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 65076
  total_bytes: 65076
}

2022-02-02 03:11:18,709 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 65076
  total_bytes: 65076
}

2022-02-02 03:11:18,961 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 65076
  total_bytes: 65076
}
local_info {
}

2022-02-02 03:11:20,600 INFO    MainThread:71 [wandb_run.py:_append_files():2194] logging synced files
